### PR TITLE
Stories parsing & login

### PR DESCRIPTION
**Pull request by zzbazza**
Added possibility to scrape IG stories from logged in session
Added cookies store for setting more then pne account for logged in scraping
Added posibility to export cookies using login password and code sent to email by login process

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
 		"@types/underscore": "^1.10.0",
 		"apify": "^0.20.2",
 		"got": "^9.6.0",
+		"http": "0.0.1-security",
```
Comment (pocesar) [inline package.json:13]: http is an internal node module, is any of the modules in dependencies using this by mistake?

Comment (zzbazza) [reply package.json]: Thanks, it was mistake.

**Comment by zzbazza** [comment]
#71 , #90

File: src/index.js [hunk 2]
```diff
@@ -33,7 +33,8 @@ async function main() {
 
     const extendOutputFunction = parseExtendOutputFunction(input.extendOutputFunction);
 
-    if (proxy.apifyProxyGroups && proxy.apifyProxyGroups.length === 0) delete proxy.apifyProxyGroups;
+    if (proxy && proxy.apifyProxyGroups && proxy.apifyProxyGroups.length === 0) delete proxy.apifyProxyGroups;
```
Comment (metalwarrior665) [inline src/index.js:36]: I think you can remove this, proxyConfiguration takes care of this.

File: src/index.js [hunk 3]
```diff
@@ -55,7 +56,7 @@ async function main() {
     }
 
     try {
-        if (!proxy) throw errors.proxyIsRequired();
+        if (Apify.isAtHome() && (!proxy || (!proxy.apifyProxyGroups && !proxy.proxyUrls))) throw errors.proxyIsRequired();
```
Comment (metalwarrior665) [inline src/index.js:59]: Instead of `proxy.apifyProxyGroups` should be `proxy.useApifyProxy`

**Pull request merged**